packages/core/src/model/chain/palm.ts

```diff
@@ -1,23 +1,42 @@
 import { Chain } from '../../constants'
+import { getAddressLink, getTransactionLink } from '../../helpers/chainExplorerLink'
+
+const palmExplorerUrl = 'https://explorer.palm.io'
 
 export const Palm: Chain = {
   chainId: 11297108109,
   chainName: 'Palm',
   isTestChain: false,
   isLocalChain: false,
   multicallAddress: '0x99a73dfE34578348fb81BD078201C0BA84E9c840',
-  getExplorerAddressLink: (address: string) => `https://explorer.palm.io/address/${address}`,
-  getExplorerTransactionLink: (transactionHash: string) => `https://explorer.palm.io/tx/${transactionHash}`,
+  rpcUrl: 'https://palm-mainnet.infura.io/v3/3a961d6501e54add9a41aa53f15de99b',
```

Comment:
Whose id is this?
If there is no public rpc without IDs, we should maybe skip the `rpcUrl` here